toolkits/global/packages/global-autocomplete/js/index.js

```diff
@@ -120,8 +123,11 @@ const autoComplete = arguments_ => {
 
 	const generateSuggestions = data => {
 		removeSuggestions();
+		data = toSuggestionsCallback(data);
```

Comment:
I would use a new reference if you are modifying the data and maybe call it something like suggestions?

```suggestion
		const suggestions = toSuggestionsCallback(data);
```